### DISCUSSION TITLE
im new after having an account for 7 years

### DISCUSSION THREAD

**Discussion by duelmas** [community/community#76387] [New to GitHub]
is Github a WYSIWYG? 

What's with all the clicking?

</samuel>

**Comment by ibtissammassa** [comment]
Hey! So, GitHub's not really the 'what-you-see-is-what-you-get' type of thing, more like the 'manage-code-and-collaborate' kind! And yeah, there's a fair bit of clicking involved—navigating through repos, checking out code, managing stuff—it's a click party, haha! But hey, if you need a hand figuring it out or want more details, just give me a shout! 😄

**Reply by duelmas** [reply]
Yeah. Seems similar to social media. I can code and started a few repositories that will just sit there unfinished in a cob web page collecting dust perhaps. This is my second account on here. Been clicking around for a while now. It's great to find open source programs I can download and run. Missing the collaborative aspect, feeling lost, alone and in the dark abyss of the internet.

**Reply by ibtissammassa** [reply]
Totally get it! Sometimes GitHub feels like a vast ocean of projects. Don't worry if things are sitting idle; it happens to the best of us. There's a ton of open-source stuff to explore. And if you're missing collaboration, There are actually a lot of communities waiting to team up. Don't hesitate to ask if you need a hand navigating this coding cosmos 😊

**Reply by duelmas** [reply]
Yes please, point me in the right direction. Where can I find interesting or inspiring communities looking for extra front end ui/ux, coders?
